js/siteorigin-panels/view/builder.js

```diff
@@ -731,8 +731,15 @@ module.exports = Backbone.View.extend( {
 	handleContentChange: function () {
 
 		// Make sure we actually need to copy content.
-		if ( panelsOptions.copy_content && this.attachedToEditor && this.$el.is( ':visible' ) ) {
-
+		if ( panelsOptions.copy_content	&&
```

Comment:
I agree with [this answer](https://softwareengineering.stackexchange.com/a/3451) on StackExchange that if an if statement gets big, it becomes unreadable. We should probably refactor it into a well named function.

Reply:
How does `isContentCopyable` or `isCopyContentEnabled` sound? If neither, do you have any suggestions?

Hm. Actually, we could go further and split this up into three functions - `isCopyContentEnabled`, `isClassicEditor`, `isBlockEditor`.

Reply:
I don't have any strict preferences here, so which ever you feel is neater works for me 🚀